client/src/components/LeafletOsmMap/SavedPlaces.js

```diff
@@ -0,0 +1,159 @@
+import {h, Component} from "preact";
+import {makeRequest} from "../../js/server-requests-utils";
+
+/**
+ * Leaflet related imports: leaflet, pouchdb module, and routing machine module
+ */
+import '../../../node_modules/leaflet/dist/leaflet.css';
+import '../../../node_modules/leaflet-routing-machine/dist/leaflet-routing-machine.css';
+import L from '../../js/leaflet-tileLayer-pouchdb-cached';
+
+/* Set all settings for saved place icon
+Waiting for new icon for favorites
+Full list of options: http://leafletjs.com/reference-1.3.0.html#icon
+ */
+const FAV_MARKER_ICON = L.icon({
+  iconUrl: '../../assets/icons/leaflet/marker-icon-fav-2x.png',
+  className: 'favorites',
+  iconSize: [25, 41], //native aspect ratio: [28, 41]
+});
+
+
+const CURRENT_USER = makeRequest('GET', 'user')
+  .catch(err=>{
+    //user not logged in
+  });
+ 
+ 
+
+export default class SavedPlaces extends Component {
```

Comment:
what is the purpose of this component? It doesn't seem to be exported nor used internally in this file.

Reply:
I created this component as a module for future use.  

If we need to import the saved places of another user on the map of the current user, you can just call <SavedPlaces user={userPromised} map={mapObj}> but I can delete it if you don't think it's necessary.

Reply:
This module was removed

Reply:
Ahhh...okay, I see where you going with that; interesting feature. I was mostly curious if I had missed something. I couldn't easily tell how it fit in the current code base. I'd recommend for similar code snippets like that adding a "TODO" comment with a quick explanation of potential use case.